cli/command/environment_test.go

```diff
@@ -401,18 +519,29 @@ func TestEnvironmentUnlink(t *testing.T) {
 			Resolve("environment", "env_name2").
 			Return([]string{"env_id2"}, nil)
 
-		req := models.DeleteEnvironmentLinkRequest{
-			SourceEnvironmentID: "env_id1",
-			DestEnvironmentID:   "env_id2",
+		base.Client.EXPECT().
+			ReadEnvironment("env_id1").
+			Return(&models.Environment{
```

Comment:
I'll add this to ROT, but if variables span multiple lines in our `gomock` chains, let's declare them outside of the chain.

Reply:
✅